src/main/java/generated/uk/gov/courtservice/schemas/courtservice/SittingStructure.java

```diff
@@ -74,7 +74,7 @@
 public class SittingStructure {
 
     @XmlElement(name = "CourtRoomNumber")
```

Comment:
We have some tech debt around the checkin of these classes into generated. They should be ideally generated from the xsd files

Reply:
Yeah I thought as much, is there a ticket for it?

I've amended the XSD as well so shouldn't cause an issue once that is done.